src/community/china/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Using Flutter in China
+description: How to use, access, and learn about Flutter in China.
+toc: true
+os-list: [Windows, macOS, Linux, ChromeOS]
+---
+
+{% assign flutter-sdk = 'flutter_opsys_v3.13.0-stable.' %}
+{% capture sdk-path -%}flutter_infra_release/releases/stable/opsys/{{flutter-sdk}}{%- endcapture %}
+
+To speed the download and installation of Flutter in China,
+consider using a [mirror site][] or _mirror_.
+
+{{site.alert.important}}
+  Use mirror sites _only_ if you _trust_ the provider.
+  The Flutter team can't verify their reliability or security.
+{{site.alert.end}}
+
+[mirror site]: https://en.wikipedia.org/wiki/Mirror_site
+
+## Use a Flutter mirror site
+
+The [China Flutter User Group][] (CFUG) maintains a Simplified Chinese
+Flutter website [https://flutter.cn](https://flutter.cn) and a mirror.
+Other mirrors can be found at the [end of this guide](#known-trusted-community-run-mirror-sites).
+
+### Configure your machine to use a mirror site
+
+To install or use Flutter in China, use a trustworthy Flutter mirror site.
+This requires setting two environment variables on your machine.
+
+_All examples that follow presume that you are using the CFUG mirror._
+
+To set your machine to use a mirror site:
+
+{% comment %} Nav tabs {% endcomment -%}
+<ul class="nav nav-tabs" id="china-os-tabs" role="tablist">
+{% for os in page.os-list %}
+{% assign id = os | downcase -%}
+  <li class="nav-item">
+    <a class="nav-link {%- if id == 'windows' %} active {% endif %}" id="{{id}}-tab" href="#{{id}}" role="tab" aria-controls="{{id}} {{id}}-dl {{id}}-pub" aria-selected="true">{{os}}</a>
+  </li>
+{% endfor -%}
+</ul>
+
+{% comment %} Tab panes {% endcomment -%}
+<div class="tab-content">
+{% for os in page.os-list %}
+{% include_relative _os-settings.md ref-os=os sdk=flutter-sdk %}
+{% endfor -%}
+</div>
+
+### Download Flutter archives based on a mirrored URL
+
+To download Flutter archives from a mirrored URL,
+replace `storage.googleapis.com` with the URL of your trusted mirror.
+This should improve download speed.
+You can use the mirror URL in other applications to download software.
+
+To change a Google storage site to a CFUG storage site, replace the example
+original URL with the example mirror URL.
```

Comment:
I'm a bit lost on what this sentence is trying to do. Is it trying to act as an instruction? Or is trying to introduce the following example? I think it would help be more explicit if it's about the following example.

```
The following example shows how to update an example Flutter download URL
from using Google's archive to using CFUG's mirror.
```